pkg/hive/cell/config.go

```diff
@@ -165,3 +166,44 @@ func stringToCIDRHookFunc(from reflect.Type, to reflect.Type, data interface{})
 	}
 	return cidr.ParseCIDR(s)
 }
+
+// stringToSliceHookFunc converts strings to []string by splitting by spaces or commas.
+func stringToSliceHookFunc(from reflect.Type, to reflect.Type, data interface{}) (interface{}, error) {
+	if from.Kind() != reflect.String || to.Kind() != reflect.Slice {
```

Comment:
nit: in my opinion, the && form of this expression is a little bit easier to read.

Reply:
Marking resolved since the func is gone now.